templates/mongodb-atlas-main.template.yaml

```diff
@@ -140,6 +140,7 @@ Parameters:
     Type: String
     Default: "5.0"
     AllowedValues:
+      - "4.2"
```

Comment:
We agreed to not remove 4.2 as it is reached EOL

Reply:
ah, I wasn't aware of this decision. I will remove 4.2 from all the templates. Thanks!

Reply:
Updated